operations/mimir-tests/test-ingest-storage-migration-step-6.jsonnet

```diff
@@ -0,0 +1,11 @@
+// Migration step 6:
+// - Decommission ingester-zone-[abc] StatefulSet and Service.
+(import 'test-ingest-storage-migration-step-5.jsonnet') {
+  _config+:: {
+    // This builds on previous step.
+    ingest_storage_migration_classic_ingesters_scale_down: false,
+    ingest_storage_migration_classic_ingesters_zone_a_decommission: true,
+    ingest_storage_migration_classic_ingesters_zone_b_decommission: true,
+    ingest_storage_migration_classic_ingesters_zone_c_decommission: true,
```

Comment:
There is only one field for scaling down all zones, but 3 fields for decommission. If these statefulset sets are scaled down already, why not decomission all of them at once?

Reply:
OK, I see that this variable is also used in later steps.

Reply:
We'll decommission them all together, but then restore 1 zone at a time when we migrate `ingester-zone-[abc]-partition` back to `ingester-zone-[abc]`.